CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
 ### Added
 
+- Export a `getDefaultInitialViewState` function for getting a deafult inital view state given a loader and desired size of view.
```

Comment:
Do you think this makes sense to expose to the public API? seems like we just call it internally if an initial view state isn't provided.

Reply:
I figure it's a nice utility for applications that use just the layers, but I can remove if you disagree - it contains some non-trivial logic that people may find helpful.